app/routes/languages.rb

```diff
@@ -6,9 +6,11 @@ class Languages < Core
       TOPICS = [:about, :exercises, :installing, :tests, :learning, :resources, :help, :launch].freeze
 
       get '/languages' do
-        tracks = X::Track.all
-        active, inactive = tracks.partition(&:active?)
-        planned, inactive = inactive.partition(&:planned?)
+        tracks = Trackler.tracks
+        active, inactive = tracks.partition { |t| t.active? }
+        planned = []
```

Comment:
> I also feel like we should be adding this functionality to Trackler

I agree that it should be in the track module and hopefully @kytrinyx can weigh in on it should be implemented..